File: Day-13/Pattern15.js
function PatternFifteen(n) {
  let i = 1;

  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= i) {
      str += String.fromCharCode(65 + i - 1);
      j++;
    }
    i++;
    console.log(str);
  }
}

PatternFifteen(4);

File: Day-13/Pattern15.py
n = int(input())
def PatternFifteen(n):
    i = 1
    while (i <= n):
        j = 1
        while(j <= i):
            print(chr(65 + i -1), end='')
            j += 1
        i += 1
        print(end='\n')


PatternFifteen(n)